web/src/main/ui/app/components/flows-new/edit-flow/ingest/ui/ingest-ui.component.ts

```diff
@@ -2,12 +2,12 @@ import {Component, EventEmitter, Input, OnInit, Output} from "@angular/core";
 
 const settings = {
   inputFilePath: {
-    label: 'Input File Path',
+    label: 'Source File Path',
```

Comment:
we need to change the selector for this element. Right now it's a class folder-path. We should change it to source-file-path